test/e2e/lifecycle/upgrade_test.go

```diff
@@ -0,0 +1,65 @@
+package test
+
+import (
+	"fmt"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+
+	. "github.com/kubevirt/cluster-network-addons-operator/test/check"
+	. "github.com/kubevirt/cluster-network-addons-operator/test/operations"
+	. "github.com/kubevirt/cluster-network-addons-operator/test/releases"
+)
+
+var _ = Context("Cluster Network Addons Operator", func() {
+	testUpgrade := func(oldRelease, newRelease Release) {
+		Context(fmt.Sprintf("when operator in version %s is installed and supported spec configured", oldRelease.Version), func() {
+			BeforeEach(func() {
+				InstallRelease(oldRelease)
+				CheckOperatorIsReady(10 * time.Minute)
+				CreateConfig(oldRelease.SupportedSpec)
+				CheckConfigCondition(ConditionAvailable, ConditionTrue, 10*time.Minute, CheckDoNotRepeat)
+				CheckReleaseUsesExpectedContainerImages(oldRelease)
+				CheckConfigVersions(oldRelease.Version, oldRelease.Version, oldRelease.Version, CheckImmediately, CheckDoNotRepeat)
+			})
+
+			Context("and it is upgraded to the latest release", func() {
+				BeforeEach(func() {
+					UninstallRelease(oldRelease)
+					InstallRelease(newRelease)
+					CheckOperatorIsReady(10 * time.Minute)
```

Comment:
Constant for 10 * time.Minute, I see it all around

Reply:
Done